src/coreComponents/common/Logger.hpp

```diff
@@ -438,7 +457,7 @@
  * @param[in] minLevel minimum log level
  * @param[in] msg a message to log (any expression that can be stream inserted)
  */
-#define GEOS_LOG_LEVEL( minLevel, msg ) GEOS_INFO_IF( this->getLogLevel() >= minLevel, msg );
+#define GEOS_LOG_LEVEL( minLevel, msg ) GEOS_LOG_IF( this->getLogLevel() >= minLevel, msg );
```

Comment:
I didn't use `GEOS_LOG_LEVEL` for this output, but I tried it at some point, which included a bunch of information about the file and line that the message was issued. This avoids the file:line output and just outputs the message.

Reply:
I think the idea was to have the line info though.

Reply:
then that is not a clean log entry??